docs/docs/guides/guidelines.md

```diff
@@ -0,0 +1,229 @@
+
+### Don't:
+
+- Judge quality based on personal beliefs. Assuming an opinion was warranted,
+  fulfills the users request and doesn't violate any guidelines, it should not
+  impact the rating of the reply.
+- Skip a label just because the reply is spam. Each label can help the model
+  improve.
+- Rate a reply's ability to answer a question correctly if you aren't certain.
+
+## 5. Classifying an initial prompt or user reply {#classifying-user}
+
+### Do:
+
+- Rate every criteria of each prompt, unless it can't be discerned because it is
+  spam or inappropriate.
+- Judge quality based on how well the prompt adheres to the guidelines.
+- Make sure to read the prompt thoroughly.
+- Use the [label explanations](#label-explanation) to determine which labels
+  apply to the prompt.
+
+### Don't:
+
+- Judge quality based on personal beliefs. The opinion of the user should not
+  impact the rating of the prompt.
+- Skip a label just because the reply is spam. Each label can help the model
+  improve.
+
+## 6. Ranking assistant replies {#ranking-assistant}
+
+### Do:
+
+- Make sure to read every available reply.
+- Think about which reply best satisfies the request of the user.
+- Rank replies based on how well they adhere to the guidelines. Factual accuracy
+  and helpfulness are first and foremost.
+- Penalize replies that fail to provide adequate warnings or caveats.
+- Penalize replies that are difficult to read due to a lack of formatting,
+  capitalization or other errors.
+- Penalize replies if the requested information is obfuscated by superfluous
+  details that make up a large part of the message.
+
+### Don't:
+
+- Rank replies based on personal beliefs. Assuming an opinion was warranted,
+  fulfills the users request and doesn't violate any guidelines, it should not
+  impact the rating of the reply.
+- Rank replies based on how long and short they are - instead, find out which
+  reply best answers the query of the user.
+
+## 7. Label Explanation {#label-explanation}
+
+**Is the message spam?**
+
+Is the message an unrelated piece of text that doesn't contribute to the
+previous prompt in any way?
+
+**Is it a bad reply, as an answer to the prompt task?**
+
+Does the assistant reply fail to engage the request of the user or does it
+contain incorrect information?
+
+### Tags
+
+**Not Target Language**: Unless specifically requested, is the language used
+different from the target language of the data set?
+
+**Not Appropriate**: In case of a user prompt, is it missing a clear request?
+
+**Contains PII**: Does the message contain personally identifying information
+(PII) that isn't publicly available and can be used to determine the identity of
+the user or a private third party?
+
+**Hate Speech**: Does the message contain prejudice towards a group classified
+as protected according to German, UK or US law?
+
+**Sexual Content**: Does the message contain sexually explicit content, such as
+descriptions of sexual acts?
```

Comment:
For prompts: shouldn't asking for smut also be covered?

Reply:
Yeah I think so. If we simply don't have prompts that ask for smut then we also won't have answers that deny it, which means that we aren't preventing people using it to make smut if they want, but we aren't supporting it either. 

This makes it a dataset issue rather than a moral stance, and we aren't trampling on any future person's libido because we don't approve.

Reply:
I mean, apart from moral issues there is also just a technical one: if the dataset has examples of the assistant saying no to something then the trained assistant will inevitably learn to incorrectly say no to things that are unproblematic. For example, the assistant saying that it can't talk about the possible risks of anal sex because that would be lewd.